homeassistant/components/drop_connect/entity.py

```diff
@@ -0,0 +1,52 @@
+"""Base entity class for DROP entities."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    CONF_DEVICE_DESC,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_OWNER_ID,
+    CONF_DEVICE_TYPE,
+    CONF_HUB_ID,
+    DEV_HUB,
+    DOMAIN,
+)
+from .coordinator import DROPDeviceDataUpdateCoordinator
+
+
+class DROPEntity(CoordinatorEntity[DROPDeviceDataUpdateCoordinator]):
+    """Representation of a DROP device entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, entity_type: str, coordinator: DROPDeviceDataUpdateCoordinator
+    ) -> None:
+        """Init DROP entity."""
+        super().__init__(coordinator)
+        if TYPE_CHECKING:
+            assert coordinator.config_entry is not None
+            assert coordinator.config_entry.unique_id is not None
+        self._attr_unique_id = f"{coordinator.config_entry.unique_id}_{entity_type}"
```

Comment:
```suggestion
        unique_id = coordinator.config_entry.unique_id
        self._attr_unique_id = f"{unique_id}_{entity_type}"
```
You're also using it once again later, but you only use this twice so its optional

Reply:
Cleaned up in 587c0b049ca883a359ce1cfae461ee5e7aa1ae06.